gpu_bdb/benchmark_runner/benchmark_config.yaml

```diff
@@ -17,3 +17,5 @@ verify_dir:
 sheet:
 tab:
 get_read_time: False
+
+run_nsys_profiling: False
```

Comment:
I wonder if we should rename this to `start_local_cluster` instead. since the nsys profiling bit comes from the way we run the query.

Reply:
Renamed the var 👍